tools/workspace-scripts.js

```diff
@@ -151,6 +151,13 @@ module.exports = {
           description: '@awarns/notifications: Build',
         },
       },
+      // @awarns/phone-sensors
+      'phone-sensors': {
+        build: {
+          script: 'nx run phone-sensors:build.all',
```

Comment:
Missing --parallel=1 flag

Reply:
--parallel=false?

Reply:
Yes, you're right. `--parallel=1` is for run-many